Automattic/jetpack
# Dialogue: move timestamp and toggle buttons out of main control component

This PR moves out two components that are defined into the `<TimestampEditControl />` component, fixing a visual blink issue in the timestamp button rendered in the canvas editor and how it works when the user tries to playback to its position. 

Fixes #

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:

* Create/edit a post
* Add a podcast block
* Create Conversation/Dialogue blocks composition
* Show the Timestamp 
* Confirm that when the player is playing, the button doesn't blink the border styles
* Confirm that it playbacks in the first click

<img width="660" alt="Screen Shot 2021-03-09 at 9 14 17 AM" src="https://user-images.githubusercontent.com/77539/110469032-df615300-80b7-11eb-8fe4-a1db733ada28.png">


#### Proposed changelog entry for your changes:


*
